bocoup/aria-at-report
# View results

This PR does a lot of things because I wanted to get in some bigger changes early on Friday so that it could be code reviewed and tests. I'm sorry it's so much at once!

1. It adds a read only view of the tests
2. It simplifies the buttons, per Matts request. There is no longer a difference between "save and go to next" and "skip and go to next test". There is now just a "next test" button.
3. When editing the code to change the buttons around, the code became nightmarish, and I have a feeling this is not the "final state" of the buttons/actions on the test run page. So I did a minor refactor to break part of the page out into it's own component.

To test this PR:
- Start a test run
- Try all the buttons except for the "issue" button and the "redo test"button
- Fill in all the data for a test and click "review test" in the iframe, then try click "next or "previous", then go back to that test and see the read only view.
